src/ophys_etl/modules/segmentation/seed/schemas.py

```diff
@@ -0,0 +1,52 @@
+import argschema
+
+
+class SeederBaseSchema(argschema.schemas.DefaultSchema):
+    fov_shape = argschema.fields.Tuple(
```

Comment:
What is the use case for specifying this by hand, as opposed to just reading it from the movie file? Is it for excluding the motion border?

Reply:
I agree, this is a little awkward, let me fix it up

Reply:
done